android/app/src/main/java/com/bounswe2020group3/paperlayer/login/LoginFragment.kt

```diff
@@ -6,20 +6,65 @@ import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
 import android.widget.Button
-import androidx.navigation.Navigation
+import android.widget.EditText
 import com.bounswe2020group3.paperlayer.R
 
-class LoginFragment : Fragment() {
+class LoginFragment : Fragment(),LoginContract.View {
+
+    //My presenter object
```

Comment:
I think you should omit "My" in comments

Reply:
Done.